LocalMultiplayerAgent/DeploymentTool/DeploymentScript.cs

```diff
@@ -0,0 +1,256 @@
+﻿using Microsoft.Azure.Gaming.LocalMultiplayerAgent.Config;
```

Comment:
nit: in general, favor alphabetizing these if you can. I don't know if we support doing system first or doing it all, but make sure you keep them in some order